hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -105,6 +107,7 @@ private AbfsClient(final URL baseUrl, final SharedKeyCredentials sharedKeyCreden
     this.userAgent = initializeUserAgent(abfsConfiguration, sslProviderName);
     this.abfsPerfTracker = abfsClientContext.getAbfsPerfTracker();
     this.abfsCounters = abfsClientContext.getAbfsCounters();
+    this.enableCorrelationHeader = abfsConfiguration.isCorrelationHeaderEnabled();
```

Comment:
AbfsClient doesnt need such a field. Comment added in AzureBlobFileSystem, where this should be an input to tracingContext init.

Reply:
Removed